src/main/java/com/mock/preparation/ds/ReverseaNumber.java

```diff
@@ -0,0 +1,19 @@
+package mock.preparation.ds;
+
+import java.util.Scanner;
+
+public class ReverseaNumber {
+    public static void main(String[] args) {
+        int num, rem = 0, rev = 0;
+        Scanner scanner = new Scanner(System.in);
+        num = scanner.nextInt();
+        while (num != 0) {
+            rem = num % 10;
+            rev = rev * 10 + rem;
+            num = num /= 10;
```

Comment:
extra = operator it will not compile. dont commit anything if not compile and output is not coming as expected. You should only commit if your testing is completed.